sds/src/proximity_keywords.rs

```diff
@@ -356,7 +370,7 @@ fn get_char_type(c: &char) -> CharType {
     }
 }
 
-fn standardize_path_chars<F>(chars: Vec<char>, mut push_character: F)
+pub fn standardize_path_chars<F>(chars: Vec<char>, mut push_character: F)
```

Comment:
As this method becomes public, it's a good practice to write some rust doc (starting with `///`) to describe what the goal of the method is.

Reply:
[Add documentation for standardize_path_chars](https://github.com/DataDog/dd-sensitive-data-scanner/pull/53/commits/8f03a2114bf2bd03e8042f6cd2354cd1ecd16792)